packages/client-python/src/rocketride/cli/commands/runtime.py

```
@@ -0,0 +1,894 @@
+After=network.target
+
+[Service]
+Type=oneshot
+ExecStart={exe} engine reconcile
+
+[Install]
+WantedBy=default.target
+"""
+        unit_path.write_text(unit_content)
+
+        subprocess.run(['systemctl', '--user', 'daemon-reload'], capture_output=True)
```

Ruff (0.15.7)

[warning] Async functions should not run processes with blocking methods

(ASYNC221)